aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -919,15 +947,31 @@ module aptos_framework::multisig_account {
         multisig_account_seed
     }
 
-    fun validate_owners(owners: &vector<address>, multisig_account: address) {
-        let distinct_owners: vector<address> = vector[];
-        vector::for_each_ref(owners, |owner| {
-            let owner = *owner;
-            assert!(owner != multisig_account, error::invalid_argument(EOWNER_CANNOT_BE_MULTISIG_ACCOUNT_ITSELF));
-            let (found, _) = vector::index_of(&distinct_owners, &owner);
-            assert!(!found, error::invalid_argument(EDUPLICATE_OWNER));
-            vector::push_back(&mut distinct_owners, owner);
-        });
+    fun validate_owners(
+        owners_ref: &vector<address>,
+        multisig_address: address
+    ) {
+        assert!(
+            !vector::contains(owners_ref, &multisig_address),
+            EOWNER_CANNOT_BE_MULTISIG_ACCOUNT_ITSELF
```

Comment:
Can you use error::invalid_argument wrapper here to be consistent as we do that everywhere in the framework?